### ISSUE TITLE
1.21.1 Support?

### ISSUE THREAD

**Issue by 5cript** [uecasm/colony4cc#32]
I hate to be that guy that asks for a new version port, but I had an automatic builder supply program (from AE2) and colony overwatch website built that relies on this mod, which I'd like to revive if possible.

**Comment by uecasm** [comment]
Yeah, I know, I definitely plan to port it, but it's a big jump and IRL is kicking my butt at the moment.

**Comment by uecasm** [comment]
This is now released.
